progressEval.html

```diff
@@ -0,0 +1,458 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+<meta charset="utf-8">
+<meta http-equiv="X-UA-Compatible">
+<meta name="viewport" content="width=device-width, initial-scale=1">
+<title>
+Course Progress Evaluator
+</title>
+<style>
+body {
+	font-family: sans-serif;
+	color: black;
+}
+input[type="text"] {
+   color: black;
+}
+
+</style>
+</head>
+<body>
+		<h2>Course Progress Evaluator</h2>	
+		<form role="form" id="grades" name="grades">	
+		<p> What week is it? </p>
+		<select class="" id="weekselect" onchange="calculate()">
+	      <option id="week" value="0">Week 0</option>
+		  <option id="week" value="1">Week 1</option>
+		  <option id="week" value="2">Week 2</option>
+		  <option id="week" value="3">Week 3</option>
+		  <option id="week" value="4">Week 4</option>
+		  <option id="week" value="5">Week 5</option>
+		  <option id="week" value="6">Week 6</option>
+	      <option id="week" value="7">Week 7</option>
+	      <option id="week" value="8">Week 8</option>
+	      <option id="week" value="9">Week 9</option>
+	      <option id="week" value="10">Week 10</option>
+	      <option id="week" value="11">Week 11</option>
+	      <option id="week" value="12">Week 12</option>
+	      <option id="week" value="13">Week 13</option>
+	      <option id="week" value="14">Week 14</option>
+	      <option id="week" value="15">Week 15</option>
+		</select>
+    <p>Fill in the table below to see your progress.</p>
+    <p>Week Number &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; Attendance &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;   Homework  &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;  Advanced Homework</p>
+		<p id="week0">Please Select a Week</p>
+    <p id="week0Alt" style="visibility: hidden;">
+      &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
+      &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
+      &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
+      Enter 0 to 3
+      &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
+      Enter 0 to 4
+      &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
+      Enter 1 if completed
+    </p>
+	<p id="week1" style="visibility: hidden;">Week 1 
+      &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; 
+      <input type="number" id="att1" onchange="calculate()" value="" placeholder="No Attendance Week 1" readonly="readonly">
```

Comment:
rather than keeping these elements `readonly`, I'd suggest disabling them instead so that users can't even interact with the input element at all.

i.e.
`<input type="number" id="att1" onchange="calculate()" value="" placeholder="No Attendance Week 1" disabled>`

This applies for every instance where you had used `readonly="readonly"`

Reply:
Also as a general note for all of your input elements, there are set values for what those points could be, so it could be a good idea to use `step`, `min`, and `max`. I've created a [codepen here](https://codepen.io/shoelaces/pen/aYOVYW) to show you what I mean